packages/live-preview-sdk/src/constants.ts

```diff
@@ -9,6 +9,7 @@ export const TOOLTIP_HEIGHT = 32;
 export const TOOLTIP_PADDING_LEFT = 5;
 
 export const MAX_DEPTH = 10;
+export const MAX_RTE_DEPTH = 2;
```

Comment:
Can you explain why we need 2 for rich text?

Reply:
1 Level depth for the direct references
Assuming it's an entry it could be relevant to resolve also the asset of this entry, therefore then 2